Firmware/xflash_layout.h

```diff
@@ -0,0 +1,50 @@
+// XFLASH memory layout
+#pragma once
+#include <stdint.h>
+#include "config.h"
+
+#define XFLASH_SIZE 0x40000ul // size of XFLASH
+#define SRAM_SIZE   0x2000u   // size of SRAM
```

Comment:
Avr libc has macros for these.

Reply:
Didn't know. RAMEND or something else?

Reply:
There are macros for flash end, sram start, sram end (RAMEND) and also start and end for XRAM.
Don’t know what the macro names are. I’m not at a pc right now. They should be located in avr/iom2560.h or something like that

Reply:
But to get access to them you include avr/io.h